src/Tizen.NUI/src/public/Common/Vector4.cs

```diff
@@ -20,6 +20,32 @@
 
 namespace Tizen.NUI
 {
+    internal class DaliVector4 : Disposable
```

Comment:
can Disposable be removed? or need ReleaseSwigCPtr from Disposable be overridden?